build/azure-pipelines/sdl-scan.yml

```diff
@@ -105,6 +105,8 @@ stages:
             env:
               VSCODE_QUALITY: stable
 
+          - template: ./common/install-builtin-extensions.yml
```

Comment:
What fails if we don't have this step ?

Reply:
I might've added this step to try and fix some old compile errors that occurred in the product. I merged in some compile error fixes on Friday and today, so I'll need to reconfirm whether this step is needed.